tomviz/python/tomviz/_internal.py

```diff
@@ -22,6 +22,7 @@ def in_application():
 
 if in_application():
     import tomviz._wrapping
+    import vtk
```

Comment:
Do we need to import all of VTK, last I checked we built VTK as modules, and we worked hard to improve start up time by just importing the modules we needed. I.e. `from vtk import vtkImageData`, a quick grep shows a few places where we have started importing more but I would prefer to get back to just the required modules.

Reply:
Okay, I think I was actually just importing it to get `vtkDataObject`. I'll change it to only get `vtkDataObject`, and I'll check to make sure I don't import all of vtk in other places in this PR.

Reply:
Fixed.